entity-services-examples/README.md

```diff
@@ -1,61 +1,81 @@
-these artifacts will be deployed to a MarkLogic server.  To change the location of the deployment
-edit `entity-services-examples/gradle.properties`
-
-
-## A Model for Races, Runs, and Runners
-
-This entity type model is very simple.  It comes complete with some source data, a model, but has some test data associated
-with it and demonstrate the various aspects of this feature.
-
-
-### Step 1 load `data/entity-types/simple-race.json` and instance data.
-
-To run a Java Application that loads all entity types, instance data, and reference data simply run
-
-When you run `gradle entity-services-examples:run` *from the parent project directory*
-
-
-There are examples in Java using the MarkLogic Data Movement SDK to load both entity type documents and instance data.
-
-src/main/resources/application.properties provides the paths to the DMSDK classes for data loading.
-
-EntityServicesLoader loads the entity-types.
-JsonInstanceLoader loads the instance data
-
-
-
-### Step 3 generate some code artifacts.
-
-This example uses QConsole to invoke entity services functions.  So pop open QConsole, import the workspace at `qc/race-workspace.xml`.
-
-The first tab has code to generate various things having to do with the entity types in `simple-race.json`.
-
-Edit the filesystem output to be usful for your checkout, then execute it.
-
-
-### Step 4 deploy modified artifacts
-
-
-`gradle mlDeploy`
-
-
-### Step 5 Use them
-
-The QConsole workspace has just a couple of SQL and SPARQL queries at this point to peruse.
-
+* A QConsole workspace `races-qc.xml` contains XQuery scripting to work with
+ the Entity Services API.
+* Examples of code that has been generated, then edited are in directories under 
+ `src/main`
+* Java classes, with source under `src/main/java` demonstrate use of the Data
+ Movement SDK for an Entity Services toolchain.
+
+*Run all commands from this directory.  The example code will not deploy to
+MarkLogic if run from the parent project's directory.*
+
+Step One. Deploy
+-----------------
+
+`../gradlew mlDeploy` deploys code to a MarkLogic server.  To change the
+location of the deployment edit `gradle.properties`
+
+* Databases and forests are provisioned as configured in `src/main/ml-config`.
+ Entity services generated the database configuration.
+* Entity Services code modules to go from `src/main/ml-modules/ext` to the
+ REST extension space.  This code was created from stubs generated by Entity 
+ Services.
+* Transforms go from `src/main/ml-modules/transforms` to REST config.  These
+ transforms wrap calls to the Entity Services modules, but were authored by
+ a developer.
+* Search options are installed from `src/main/ml-modules/options` to REST.
+ This configuration was generated by Entity Services.
+* A module to generate code is installed from
+ `src/main/ml-modules/services` to REST.  This extension is just used by the
+ example ode to invoke the Entity Services API as a demonstration.
```

Comment:
do you mean "example code"? I think it is better to explicitly call out that its authored by a developer here as well, although "demonstration" might suggest that.